SETUP.md

```diff
@@ -0,0 +1,95 @@
+# AutoSpotting Setup #
+
+It's relatively easy to build and install your own version of this tool's binaries, 
+removing your dependency on the author's version, and allowing any customizations
+and improvements your organization needs. You'll need to set up a local environment
+to run Go, compile the binaries locally, upload them to an S3 bucket in your AWS
+account, and update the CloudFormation stack to use those new binaries.
+
+## Compiling the binaries locally ##
+
+1. Install Go via the binaries [here](https://golang.org/dl/) or Homebrew:
+
+   `brew install go`
```

Comment:
Mac dependant, but if people come here, I assume they already have Go installed or know how to use a package manager.